packages/qrcode-modal/src/browser.ts

```diff
@@ -23,32 +24,30 @@ function formatQRCodeModal (qrCodeImage: string) {
   return `
     <div
       id="walletconnect-qrcode-modal"
-      style="${style.qrcode.base}"
-      class="animated fadeIn"
+      class="qrcode__base animated fadeIn"
     >
-      <div style="${style.modal.base}">
-        <div style="${style.modal.header}">
-          <img src="${asset.logo}" style="${style.modal.headerLogo}" />
-          <div style="${style.modal.close.wrapper}">
+      <div class="modal__base">
```

Comment:
it's probably best to prefix classnames with `walletconnect` to not conflict with other stylesheets